cloudinit/net/openbsd.py

```diff
@@ -0,0 +1,38 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+
+from cloudinit import log as logging
+from cloudinit import util
+import cloudinit.net.bsd
+
+LOG = logging.getLogger(__name__)
+
+
+class Renderer(cloudinit.net.bsd.BSDRenderer):
+
+    def write_config(self):
+        for device_name, v in self.interface_configurations.items():
+            if_file = 'etc/hostname.%s' % device_name
+            fn = util.target_path(self.target, if_file)
+            content = 'dhcp\n'
+            if isinstance(v, dict):
+                content = "inet {address} {netmask}\n".format(
+                            address=v.get('address'),
```

Comment:
If both of these aren't present, do we really want to produce "inet None None\n"?